acceptance/tests/pxp-module-puppet/run_puppet_result_changed.rb

```diff
@@ -1,13 +1,14 @@
 require 'pxp-agent/test_helper.rb'
-
-ENVIRONMENT_NAME = 'changed_result'
+require 'puppet/acceptance/environment_utils'
 
 test_name 'C93062 - Run puppet and expect \'changed\' result' do
+  extend Puppet::Acceptance::EnvironmentUtils
+
+  env_name = test_file_name = File.basename(__FILE__, '.*')
+  environment_name = mk_tmp_environment(env_name)
 
   step 'On master, create a new environment that will result in Changed' do
-    environmentpath = master.puppet['environmentpath']
-    site_manifest = "#{environmentpath}/#{ENVIRONMENT_NAME}/manifests/site.pp"
-    on(master, "cp -r #{environmentpath}/production #{environmentpath}/#{ENVIRONMENT_NAME}")
+    site_manifest = "#{environmentpath}/#{environment_name}/manifests/site.pp"
     create_remote_file(master, site_manifest, <<-SITEPP)
 node default {
   notify {'Notify resources cause a Puppet run to have a \\'changed\\' outcome ☃':}
```

Comment:
I didn't notice that, sorry.
This case might warrant having its own test case; so in the event of a test run failing we have 1 test result for a Changed puppet run and a separate test result for a Puppet run with non-ASCII characters.
On the other hand; a single test is more efficient in CI.

We can reject this PR; but we should at least update the test case to advertise that it is testing non-ASCII